modules/lsp/src/main/scala/generated/enumerations.scala

```diff
@@ -228,9 +254,38 @@ object SymbolKind extends UIntEnum[SymbolKind]:
     File, Module, Namespace, Package, Class, Method, Property, Field, Constructor, Enum, Interface, Function, Variable, Constant, String, Number, Boolean, Array, Object, Key, Null, EnumMember, Struct, Event, Operator, TypeParameter
   )
 
+  extension (self: SymbolKind) def name: String = (self.value: @switch) match {
```

Comment:
This one right here is why I tried to make it a switch.

Turns out, on 3.3.3 I was getting warnings on some of these, saying the compiler couldn't generate one - hence the 3.3.5 bump